src/main/fc/runtime_config.c

```diff
@@ -34,7 +34,7 @@ static uint32_t enabledSensors = 0;
 const char *armingDisableFlagNames[]= {
     "FS", "ANGLE", "CAL", "OVRLD", "NAV", "COMPASS",
     "ACC", "ARMSW", "HWFAIL", "BOXFS", "KILLSW", "RX",
-    "THR", "CLI", "CMS", "OSD"
+    "THR", "CLI", "CMS", "OSD", "LAUNCH"
```

Comment:
I think we should call this differently. Maybe `ROLL/PITCH` ?

Reply:
I hesitated between the two and finally chose LAUNCH because it was shorter but you are right ROLL/PITCH is better describing the issue.